include/boost/gil/image_processing/morphology.hpp

```diff
@@ -0,0 +1,285 @@
+//
+// Copyright 2021 Prathamesh Tagore <prathameshtagore@gmail.com>
+//
+// Use, modification and distribution are subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#ifndef BOOST_GIL_IMAGE_PROCESSING_MORPHOLOGY_HPP
+#define BOOST_GIL_IMAGE_PROCESSING_MORPHOLOGY_HPP
+#include <boost/gil/extension/numeric/kernel.hpp>
+#include <boost/gil/gray.hpp>
+#include <boost/gil/image_processing/threshold.hpp>
+
+namespace gil = boost::gil;
+namespace boost{
+    namespace gil{
+        enum class morphological_operations
+            {
+                dilation,
+                erosion,
+            };
```

Comment:
Please, don't indent content of namespaces - we don't need empty column space on the left :)